src/components/WebexMember/WebexMember.jsx

```diff
@@ -28,33 +24,18 @@ import webexComponentClasses from '../helpers';
  */
 export default function WebexMember({
   className,
-  destinationID,
-  destinationType,
+  // eslint-disable-next-line no-unused-vars
+  roomID,
```

Comment:
If the room ID is not needed, something is off with this implementation...

Reply:
I think what is missing is a check that the given person actually has a membership to the room